pkg/ccl/backupccl/split_and_scatter_processor.go

```diff
@@ -95,13 +98,46 @@ func (s dbSplitAndScatterer) split(
 		newSplitKey = splitAt
 	}
 	log.VEventf(ctx, 1, "presplitting new key %+v", newSplitKey)
-	if err := s.db.AdminSplit(ctx, newSplitKey, expirationTime); err != nil {
+	if err := s.adminSplitWithRetry(ctx, newSplitKey, expirationTime); err != nil {
 		return errors.Wrapf(err, "splitting key %s", newSplitKey)
 	}
 
 	return nil
 }
 
+func (s dbSplitAndScatterer) adminSplitWithRetry(
+	ctx context.Context, newSplitKey roachpb.Key, expirationTime hlc.Timestamp,
+) error {
+
+	retryOpts := retry.Options{
+		MaxBackoff: 1 * time.Second,
+		MaxRetries: 5,
+	}
+
+	var err error
+	for r := retry.StartWithCtx(ctx, retryOpts); r.Next(); {
+
+		if err == nil {
+			break
+		}
+		err := s.db.AdminSplit(ctx, newSplitKey, expirationTime)
```

Comment:
Is it worth moving this retry logic one level deeper? Maybe we want to ask KV but don't we have a lot of callers outside restore that use `db.AdminSplit` as well? import, index backfills etc.

Reply:
Or atleast auditing all the places we call `db.AdminSplit` and deciding whether to switch them to this wrapper method.

Reply:
Good point. I don't see why the caller needs to handle these ephemeral retries. Lemme ask kv what they think in slack.